tdrs-backend/tdpservice/reports/migrations/0001_initial.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 3.1.1 on 2020-12-14 19:08
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ("stts", "0002_auto_20200923_1809"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ReportFile",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("original_filename", models.CharField(max_length=256)),
+                ("slug", models.CharField(max_length=256, unique=True)),
+                ("extension", models.CharField(default="txt", max_length=8)),
+                (
+                    "quarter",
+                    models.CharField(
+                        choices=[
+                            ("Q1", "Q1"),
+                            ("Q2", "Q2"),
+                            ("Q3", "Q3"),
+                            ("Q4", "Q4"),
+                        ],
+                        max_length=16,
+                    ),
+                ),
+                ("year", models.CharField(max_length=16)),
```

Comment:
When would we expect this to be more than 4 chars? Would this be better as an integer?

Reply:
I recall it did not like numbers smaller than 16, not sure why I didn't do it as an integer, it would take up less space. We just never use it like a number.

Reply:
Hmmm, I'm noticing that "quarter" and "section" have data integrity guards on them via `choices`. Could modeling year as an integer could add a layer of data validation? E.g. as-is, `"apple"` and `"banana"` are valid years.